.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,66 +1,32 @@
-<!--
-  ☝️How to write a good PR title:
-  - Prefix it with [ComponentName] (if applicable), for example: [Button]
-  - Start with a verb, for example: Add, Delete, Improve, Fix…
-  - Give as much context as necessary and as little as possible
-  - Prefix it with [WIP] while it’s a work in progress
--->
+<!-- PR title: [ComponentName] *Verb*ing a ... -->
 
 ### WHY are these changes introduced?
 
-Fixes #0000 <!-- link to issue if one exists -->
-
-<!--
-  Context about the problem that’s being addressed.
--->
+Fixes #0000
 
 ### WHAT is this pull request doing?
 
-<!--
-  Summary of the changes committed.
+Before / after screenshots are appreciated for UI changes.
+Include alt text that describes the screenshot.
+Put any gifs inside a `<details>` block.
 
-  Before / after screenshots are appreciated for UI changes. Make sure to include alt text that describes the screenshot.
+### Playground code
 
-  If you include an animated gif showing your change, wrapping it in a details tag is recommended. Gifs usually autoplay, which can cause accessibility issues for people reviewing your PR:
+<details>
+<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>
 
-    <details>
-      <summary>Summary of your gif(s)</summary>
-      <img src="..." alt="Description of what the gif shows">
-    </details>
--->
+```jsx
+*** Add the code you want to test in here ***
+```
 
-## <!-- ℹ️ Delete the following for small / trivial changes -->
+</details>
 
 ### How to 🎩
 
 🖥 [Local development instructions](https://github.com/Shopify/polaris-react/blob/master/README.md#development)
 🗒 [General tophatting guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md)
 📄 [Changelog guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Versioning%20and%20changelog.md)
 
-<!--
```

Comment:
We def don't want to remove this Playground part, it's a rare occurance for a PR in this repo to not require playground code be provided.

Reply:
I just moved it up to its own section above the boilerplate documentation and checklist